packages/filebrowser-extension/src/index.ts

```diff
@@ -319,8 +319,7 @@ function activateBrowser(
 
 function activateShareFile(
   app: JupyterFrontEnd,
-  factory: IFileBrowserFactory,
-  resolver: IWindowResolver
```

Comment:
This also needs to be removed from the plugin `requires` above.

Reply:
Good point.